static/scss/card.scss

```diff
@@ -5,5 +5,23 @@
 
   .content {
     margin: 20px;
+
+    .row {
+      margin: 30px 10px;
+      display: flex;
+      flex-direction: row;
+
+      div,button {
+        margin: 5px;
+      }
+
+      .active {
+        font-weight: 700;
+      }
+
+      &.post-types>div {
+        cursor: pointer;
+      }
+    }
```

Comment:
this looks like styling that's specific to the the usage for creating posts - can we move that to a separate `.scss` file so we can keep the stying for `<Card />` a little more general?